src/main/java/lotto/Input.java

```diff
@@ -0,0 +1,79 @@
+package lotto;
+
+import lotto.CustomException.WrongInputNumberException;
+
+import java.util.Scanner;
+
+public class Input {
+    private static Scanner scanner = new Scanner(System.in);
+    private static String numberRegularExpression = "^[0-9]+$";
+    public static int inputMoney(){
+        System.out.println("구입금액을 입력해 주세요.");
+        String inputValue = scanner.nextLine();
+        return convertToInteger(inputValue);
+    }
+
+    private static int convertToInteger(String inputValue){
+        String numberRegular_Expression = "^[0-9]+$";
+        if(!inputValue.matches(numberRegular_Expression)) throw new WrongInputNumberException("숫자를 입력해주세요.");
+        return Integer.parseInt(inputValue);
+    }
+
+    public static Integer[] inputPreviousLottoNumber(){
+        System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+        Integer[] prevLottoNumber = prevLottoNumberSplit(scanner.nextLine());
+        previousLottoNumberValidation(prevLottoNumber);
+        return prevLottoNumber;
+    }
+
+    public static Integer[] prevLottoNumberSplit(String previousLottoNumber){
+        Integer[] integerNumber = new Integer[6];
+        previousLottoNumber = previousLottoNumber.replace(" ","");
+        String[] splitNumber = previousLottoNumber.split(",");
+        for(int i =0;i<splitNumber.length;i++){
+            integerNumber[i] = Integer.parseInt(splitNumber[i]);
+        }
+        return integerNumber;
```

Comment:
이거는 참고만 해주세요.
```suggestion
        previousLottoNumber = previousLottoNumber.trim();
        String[] splitNumbers = previousLottoNumber.split(",");
        return Arrays.stream(splitNumbers)
            .map(splitNumber -> Integer.parseInt(splitNumber))
            .collect(Collections.toList());
```
스트림을 사용하면 List로 편안하게 바꿀 수 있는데... List를 좋아해보는건 어떨까요?